packages/launchpad/src/setup/EnvironmentSetup.vue

```diff
@@ -166,4 +218,5 @@ const canNavigateForward = computed(() => {
   return bundler !== null && framework !== null
 })
 
+const isAlertOpen = ref(erroredFrameworks.value.length > 0)
```

Comment:
Thinking about the "init delay" issues I ran into elsewhere in launchpad, should this be a `computed` instead of a `ref` in case the `erroredFrameworks` ends up resolving late and is updated via the subscription being added under my PR?

Reply:
yeah this is a good call, might as well be computed

Reply:
yeah I'll revisit, was just looking at the same thing. `computed` can have `get` and `set` handlers which make them work with v-model but it's the same general reactivity pattern as `ref` + `watchEffect`.

Reply:
I suspect there's actually a simpler way through this - the `model` is really _user_ state, it should initialize to true always, and we could `v-if` the component based on the computed value coming out of graphql. So we avoid needlessly rendering this Alert component at all if it's never going to display.

Reply:
Updated to what I described above - `v-if` for the alert itself based on a computed from gql, and a seperate `ref` for the open/closed state.